### PR TITLE
ref(ui): New message for performance shared group badge

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Review by scttcper — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]

File: static/app/views/sharedGroupDetails/sharedGroupHeader.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const SharedGroupHeader = ({group}: Props) => (
             <FeatureBadge
               type="beta"
               title={t(
-                'Sharing Performance Issues is not yet full featured and may be missing context'
+                'Not all features have been implemented for sharing Performance Issues and may be missing context.'
```
Comment (imatwawana) [inline static/app/views/sharedGroupDetails/sharedGroupHeader.tsx:33]: May I suggest:
```suggestion
                'Sharing Performance Issues hasn't been implemented for all features yet and may be missing context.'
```

Comment (evanpurkhiser) [reply static/app/views/sharedGroupDetails/sharedGroupHeader.tsx:33]: I think this isn't quite right, this wording makes it sound like sharing is something each "feature" can do, where it's more that not all issue feature related to performance issues are enabled yet for shared performance issues

Comment (imatwawana) [reply static/app/views/sharedGroupDetails/sharedGroupHeader.tsx:33]: Ah yeah, "shared" vs "sharing" makes a pretty big difference to the meaning of this. Perhaps:
```suggestion
                'Not all features have been implemented for shared Performance Issues and those issues may be missing context.'
```

Comment (evanpurkhiser) [reply static/app/views/sharedGroupDetails/sharedGroupHeader.tsx]: Thanks @imatwawana !

**Pull request merged**